Example/Backpack/View Controllers/Bottom Sheet/BottomSheetViewController.swift

```diff
@@ -62,11 +64,42 @@ extension BottomSheetViewController {
             let sheet = BottomSheet(contentViewController: wrappedContent,
                                     scrollViewToTrack: content.tableView,
                                     bottomSectionViewController: wrappedBottomSection)
+            
+            bottomSection.buttonText = "Dismiss"
 
-            bottomSection.dismissClosure = {
+            bottomSection.buttonClickedClosure = {
                 sheet.viewControllerToPresent.dismiss(animated: true, completion: nil)
             }
 
+            sheet.present(in: self, animated: true, completion: nil)
+        case internalNavigationBottomSheet:
```

Comment:
Can we change this story a bit? We don't want to encourage doing endless stacked presentations which it currently does with the persistent "next step" button. Instead can we call the story something like "Bottom sheet presenting bottom sheet" and limit to a stack of 2. 

For longer flows we want to encourage using a `UINavigationController` inside the bottom sheet.